sdk/simulator-service/src/strategies/refinance/RefinanceLendingToLending.ts

```diff
@@ -106,6 +104,27 @@ export async function refinaceLendingToLending(
         borrowTargetType: TokenTransferTargetType.PositionsManager,
       },
     }))
+    // TODO: Implement swapping logic properly. Current implementation is just placeholder
```

Comment:
What needs to be done for this? Just to understand what is left

Reply:
Everything in this PR https://github.com/OasisDEX/summerfi-monorepo/pull/153

Basically that PR implement a strategy that handles two swaps